tests/lib/config/plugins.js

```diff
@@ -85,6 +85,12 @@ describe("Plugins", function() {
             assert.deepEqual(Rules.get("example/qux"), plugin.rules.qux);
         });
 
+        it("should throw an error when a plugin has whitespace", function() {
+            assert.throws(function() {
+                StubbedPlugins.load("whitespace ");
+            }, /Whitespace found in plugin name/);
+        });
+
```

Comment:
Should we have some tests for plugin names with tab characters or newline characters as well?